freeipa-client/src/main/java/com/sequenceiq/freeipa/client/operation/GroupAddMemberOperation.java

```diff
@@ -0,0 +1,74 @@
+package com.sequenceiq.freeipa.client.operation;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.freeipa.client.FreeIpaChecks;
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.client.FreeIpaClientException;
+import com.sequenceiq.freeipa.client.model.Group;
+import com.sequenceiq.freeipa.client.model.RPCResponse;
+
+public class GroupAddMemberOperation extends AbstractFreeipaOperation<Group> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GroupAddMemberOperation.class);
+
+    private String group;
+
+    private Collection<String> users;
+
+    private BiConsumer<String, String> warnings;
+
+    private GroupAddMemberOperation(String group, Collection<String> users, BiConsumer<String, String> warnings) {
+        this.group = group;
+        this.users = users;
+        this.warnings = warnings;
+    }
+
+    public static GroupAddMemberOperation get(String group, Collection<String> users, BiConsumer<String, String> warnings) {
```

Comment:
I don't understand why do you use a static method to create an instance instead of the constructor. I don't see any additional value and also `get` is a bit misleading, `create` would be more meaningful in my opinion.

Reply:
i like to use static method for initialization to increase readability, not a big deal. yes, `create` makes more sense, i'll change these